Vagrantfile

```diff
@@ -0,0 +1,93 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+
+  config.vm.synced_folder ".", "/pgcli"
+
+  pgcli_version = ENV['version']
+  pgcli_description = "Postgres CLI with autocompletion and syntax highlighting"
+
+  config.vm.define "debian" do |debian|
+    debian.vm.box = "chef/debian-7.8"
+    debian.vm.provision "shell", inline: <<-SHELL
+    echo "-> Building DEB on `lsb_release -s`"
+    sudo apt-get update
+    sudo apt-get install -y libpq-dev python-dev python-setuptools rubygems
+    sudo easy_install pip
+    sudo pip install virtualenv virtualenv-tools
+    sudo gem install fpm
+    echo "-> Cleaning up old workspace"
+    rm -rf build
+    mkdir -p build/usr/share
+    virtualenv build/usr/share/pgcli
+    build/usr/share/pgcli/bin/pip install -U pip distribute
+    build/usr/share/pgcli/bin/pip uninstall -y distribute
```

Comment:
@ceocoder Can you please help me understand why you pip install distribute and then immediately uninstall distribute?